server/dvv.js

```diff
@@ -34,7 +34,7 @@ var DATA = [1, 2, 3];
 var PARTITION_LENGTH = 1;
 
 //Callback to be made on complete of entire distributed task
-var CALLBACK = function(results){ console.log(results); return results; };
+	var CALLBACK = function(results){ /*console.log(results); */ return results; };
```

Comment:
@jyeg As a suggestion, it would be better to keep the original `CALLBACK` definition, and in server.js, pass in a function for returning results in `dvv.config()`.

Example:
```
dvv.config({
  staticPath: '/../client',
  timeout: 25000,
  data: createMatrixArrays(200, 10),
  func: 'math.inv',
  clock: true,
  callback: function(results) { return results; }
});
```